source/_docs/wordpress-phpstorm.md

```diff
@@ -1,64 +1,97 @@
-2. Within your PHPStorm project, go to Tools > Deployment > Configuration.
+2. With your PhpStorm project opened, go to the **Tools Menu** > **Deployment** and click the **Configuration...** option.
 3. Click **+** to [add a web server](https://www.jetbrains.com/help/phpstorm/add-server-dialog.html).
 4. Enter your Pantheon site name, followed by a dash and the environment.
 5. Select **SFTP** for type and click **OK**.
- ![Add web server PHPStorm](/source/docs/assets/images/add-web-server-phpstorm.png)
+![Add web server PhpStorm](/source/docs/assets/images/add-web-server-phpstorm.png)
+
 ### Connection Tab
 
-Using your site's [SFTP connection information](/docs/sftp#sftp-connection-information), configure the deployment connection and click **Test SFTP Connection**.
+Use your site's [SFTP connection information](/docs/sftp#sftp-connection-information) to configure the Connection tab:
+
+1. Type: `SFTP`
+1. SFTP Host: `appserver.dev.<Site UUID>.drush.in`
+1. Port: `2222`
+1. User name: `dev.<Site UUID>`
+1. Auth Type: Select Password and specify password, or key pair and select your private SSH key.
+1. Root Path: Click the **Autodetect** button and the Root Path should be automatically populated:
+
+    ![Add web server PhpStorm](/source/docs/assets/images/phpstorm-deployment-connection-tab.png)
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p markdown="1">PhpStorm does not allow the Root Path to be left blank. Because the Pantheon platform sometimes migrates sites across app servers without warning, you will need to update the root path any time the binding path changes for your Dev environment.</p>
+    </div>
+
+1. Click the **Test SFTP Connection...** button and make sure the connection is successful before moving forward.  Click **OK**.
+
+    ![Add web server PhpStorm](/source/docs/assets/images/phpstorm-deployment-connection-test-sftp-success.png)
 
 ### Mappings Tab
+
 Provide the file paths within the Mappings tab to allow correspondence between project folders:
 
 1. Local path: This field is automatically populated and requires no edits.
-2. Deployment path on server: `/code/wp-content/plugins`
-3. Web path on server: URL of your Pantheon Dev environment. Example: https://dev-yoursite.pantheon.io
+2. Deployment path on server: `/code`
+3. Web path on server: `/`
 4. Click **OK**.
 
-Now go to Tools > Deployment > Automatic Upload > and select the server you created. Any files you change and save locally will be automatically uploaded to Pantheon.
+## Enabling Automatic Upload
+
+Go to **Tools Menu** > **Deployment** > **Automatic Upload**.
+
+You are now ready to begin editing your code. Any files you change and save locally will be automatically uploaded to your Pantheon development environment.Once your code is complete, you can deploy your work into Test and then ultimately to Live from your Pantheon Dashboard.
```

Comment:
We may need to change this conclusion based on the overview rewrite and the clarified objective. If the purpose of this guide is to set up a site to be used for general development, and it's not a site intended to be launched, it wouldn't align with the conclusion here.